projects/netgrif-components-core/src/assets/i18n/de.json

```diff
@@ -306,6 +306,12 @@
             "sign": "Registrieren",
             "enterPass": "Ihre Kennwort eingeben"
         },
+        "changePassword": {
+            "oldPassword": "Geben Sie Ihr altes Passwort ein",
```

Comment:
`Passwort` - `Kennwort` should be consistent across all german translations.

Reply:
Done